python/google/protobuf/internal/well_known_types.py

```diff
@@ -143,6 +144,9 @@ def FromJsonString(self, value):
     Raises:
       ValueError: On parsing problems.
     """
+    if not isinstance(value, str):
+      raise json_format.ParseError(
```

Comment:
ValueError should good enough, to keep the same with the rest of the file, and could not import google.protobuf.json_format

Reply:
Good call. I didn't notice the rest of it used ValueError.